src/helics/application_api/Inputs.cpp

```diff
@@ -215,7 +216,8 @@ size_t Input::getStringSize()
         if (np.name.empty()) {
             return 30;  //"~length of #invalid" string +20
         }
-        //+20 is just in case the converted string is actually being requested in which case it the 20 accounts
+        //+20 is just in case the converted string is actually being requested in which case it the
+        //20 accounts
         // for the string representation of a double
```

Comment:
```suggestion
        // +20 is just in case the converted string is actually being requested in which case the
        // +20 accounts for the string representation of a double
```